ShopLens/ShopLensForms/ShopLens.cs

```diff
@@ -57,21 +57,26 @@ private void CommandRecognized_WhatIsThis(object sender, EventArgs e)
         {
             if (live_video.Image != null)
             {
-                Bitmap image = (Bitmap)live_video.Image.Clone();
-                capture_picture.Image = (Bitmap)live_video.Image.Clone();
+                var image = (Image)live_video.Image.Clone();
+                capture_picture.Image = image;
 
                 var ms = new MemoryStream();
                 image.Save(ms, System.Drawing.Imaging.ImageFormat.Jpeg);
 
                 var classificationResults = _imageClassifying.ClassifyImage(ms.ToArray());
 
                 var resultStrings = classificationResults.Select(pair => $"{pair.Key} - {(int)(pair.Value * 100)} percent.");
-                _textVoicer.SayMessage("My estimates on the image are: ");
-                foreach (var result in resultStrings)
-                {
-                    _textVoicer.SayMessage(result);
-                    Thread.Sleep(500);
-                }
+                _textVoicer.SayMessage("This is");
+
+                //Order by probability values and take the first label name.
+                string mostConfidentResult = classificationResults.OrderByDescending(x => x.Value).FirstOrDefault().ToString();
+
+                //Format string so only the name of the product is said.
+                string[] results = mostConfidentResult.Split(',');
+
+                _textVoicer.SayMessage(results[0]);
```

Comment:
That's a very interesting way of doing things 😄 

How about:
```
// Since the results look like <'tomato', 0.99>, we can just take the Key
var prediction = classificationResults.OrderByDescending(x => x.Value).FirstOrDefault().Key;
if (prediction == null)
{
    // don't forget to handle null !
}
else
 {
    _textVoicer.SayMessage(prediction);
}
```

Reply:
>>compiler tells you this ain't a string
>>you write .ToString to it
>>compiler doesn't complain no more, but code is sloppy
>>??!!??
>>PROFIT!
>>then you figure out how to make it so that only the name of the product is said
>>you split the string
>>it says the name of the product
>>and in the process of all of this I didn't get that I can just get the key